scripts/build_desktop_app_with_firebase.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env python
```

Comment:
glint offers about a hundred complaints about formatting on this file. Mostly lines too long and inconsistent string delimiters (single vs double quotes). Copying the file into cider is probably the easiest way to see them.

Reply:
Thanks. Super super helpful! Will be doing this for any python development in the future. I wonder if vscode has something similar to this.

Reply:
https://g3doc.corp.google.com/devtools/gpylint/g3doc/editor_integration.md?cl=head#vscode

Appears to be the case.